WhlgPublicWebsite/Views/Partials/LocalAuthorityMessages/Confirmation/PortsmouthCityCouncil.cshtml

```diff
@@ -0,0 +1,20 @@
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model WhlgPublicWebsite.Models.Questionnaire.ConfirmationViewModel
+
+<p class="govuk-body">
+    <strong>@Model.LocalAuthorityName's official delivery partner: Warmer Homes</strong> will process
+    your details and should contact you within 10 working days.
+<p class="govuk-body">
+
+<p class="govuk-body">
+    @if (Model.LocalAuthorityIsLive)
+    {
+        @await Html.GovUkInsetText(new InsetTextViewModel
+        {
+            Text = "If you do not receive a response within 10 working days, please contact Warmer Homes directly."
+        })
+    }
+</p>
+
+<p class="govuk-body">To enquire about your application visit <a class="govuk-link" href="@Model.LocalAuthorityWebsite">https://www.warmerhomes.org.uk</a>.</p>
```

Comment:
It looks a bit strange to not have the href looking at a variable and the text to be static text. Could we use the variable for both here? (just in case somehow the variable is updated, and the text mismatches!)